File: Models/Installment.cs
﻿using System;

namespace Peohe.Models
{
    public class Installment
    {
        public int InstallmentId { get; set; }
        public double Amount { get; set; }
        public bool? Paid { get; set; }
        public DateTime DueDate { get; set; }
        public DateTime? PayDay { get; set; }
        public DateTime? Deleted { get; set; }
        public string Historic { get; set; }
        public Attendance Attendance { get; set; }
    }
}

File: ClientApp/src/app/models/Installment.ts
import { Attendance } from "../attendance/attendance";

export class Installment {
    installmentId: string;
    amount: number;
    paid?: boolean;
    installmentNumber: number;
    dueDate: string;
    payDay?: string;
    deleted?: string;
    historic: string;
    attendance: Attendance;
    aplicationUserId: string;
}